src/sardana/taurus/qt/qtgui/extra_sardana/showscanonline.py

```diff
@@ -102,7 +102,7 @@ def main(group, taurus_log_level, door):
     from taurus.qt.qtgui.application import TaurusApplication
 
     app = TaurusApplication(app_name='Showscan Online', org_domain="Sardana",
-                            org_name="Tango communinity", cmd_line_parser=None)
+                            org_name="Tango communinity")
```

Comment:
Fine for me as it is, but maybe it is better to leave it explicit in order not to make it incompatible with taurus 4.5

Reply:
Good point. I will revert this commit and then merge it myself. Thanks!